embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/session/PayloadMessageCollator.kt

```diff
@@ -172,4 +190,97 @@
             spans = spans
         )
     }
+
+    /**
+     * Creates a background activity stop message.
+     */
+    fun createStopMessage(
```

Comment:
Is there still a "message" that's not "stop" now?

Reply:
Renamed function